resources/css/bem/team-members.less

```diff
@@ -2,21 +2,36 @@
 // See the LICENCE file in the repository root for full licence text.
 
 .team-members {
-  border-radius: @border-radius--large;
   display: grid;
-  gap: 5px;
-  padding: 5px;
-  background-color: hsl(var(--hsl-b2));
+  gap: 10px;
 
-  &--owner {
-    background-color: hsl(var(--hsl-orange-1));
-    color: hsl(var(--hsl-orange-4));
+  &__type {
+    border-radius: @border-radius--large;
+    display: grid;
+    gap: 5px;
+    padding: 5px;
+    background-color: hsl(var(--hsl-b2));
+    grid-template-columns: var(--grid-template-columns);
+    --grid-template-columns: 1fr;
+    --grid-template-columns-desktop: 1fr 1fr 1fr;
```

Comment:
`repeat(auto-fill, minmax(@user-card-width-min, 1fr))`
Needs a min width to stop clipping too many icons on card:
![image](https://github.com/user-attachments/assets/34bf6316-11f8-4858-a407-7587af692c3a)

Reply:
maybe get rid of `@user-card-width-min` and update the friends page as well since there's only 10px difference with `@user-card-width` and there's more icons than ever on the card now 🤔